smartapps/juano2310/jawbone-up-connect.src/jawbone-up-connect.groovy

```diff
@@ -328,7 +320,7 @@ def setup() {
 
 def installed() {
 	enableCallback()
```

Comment:
I'm not sure that we need to make this call anymore.  Its effectively doing the same thing as /oauth/initialize except it won't work across shards.